### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by AGS-**
Added prettier to the repo and ran everything through it. I currently set some of the eslint rules to be warnings since prettier doesn't have a lot of configuration options yet. This shouldn't really matter since we're going to have consistent formatting anyway, however I would like to add these back in as soon as we can.

### CODE REVIEW

File: resources/pretty.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+];
+const glob = '{packages/*/{example,resources,src},example,resources,src}/**/*.js';
+const root = join(__dirname, '..');
+const executable = join(root, 'node_modules', '.bin', 'prettier');
+
+const check = process.argv.indexOf('--check') !== -1;
+const mode = check ? '--list-different' : '--write';
+process.chdir(root);
+
+// eslint-disable-next-line
```
Comment (wincent) [inline resources/pretty.js]: What's the lint that's firing here? Would be nice to mention the specific rule.

File: resources/pretty.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  if (check) {
+    print(`\n${YELLOW}The files listed above are not correctly formatted.`);
+    print(`Try: ${INVERSE} npm run pretty ${RESET}`);
+  }
+}
+if (error) {
+  print('error', error);
+}
+// eslint-disable-next-line
+process.exit(status != null ? status : 1);
```
Comment (wincent) [inline resources/pretty.js]: Same here, what's the lint? I wonder what is different about our `.eslintrc` files between repos that I didn't have any lints in this file when I originally wrote it [here](https://github.com/graphql/graphql-language-service/blob/master/resources/pretty.js).

**Comment by wincent** [comment]
Cool! One thing to bear in mind about landing this: it will require pretty much every pending PR to be rebased/redone, so we should have a look through them and see if there's anything we should/can get in before this one.

Why exclude the test files? If we're going to remove code-formatting as something-we-have-to-think-about, it would be nice to do that everywhere.

File: resources/pretty.js (hunk 1 above)
Comment (AGS-) [inline resources/pretty.js]: This one fires 'no-unused-vars' because of the stderr

File: resources/pretty.js (hunk 2 above)
Comment (AGS-) [inline resources/pretty.js]: This one fires eqeqeq

**Comment by AGS-** [comment]
Yeah, makes sense. I'll go through the PR's and wait a bit before landing this. I excluded the test files because it was unminifying the vendor tests, but I could just have it exclude ./test/vendor.

**Comment by AGS-** [comment]
I'm abandoning this because it's way too aggressive, I have a much better way to go about this that I'll set up in another PR

**Pull request closed without merging**